src/components/Menu/entries.js

```diff
@@ -322,6 +323,16 @@ export default [
 		},
 		priority: 17,
 	},
+	{
```

Comment:
@jancborchardt Keeping https://github.com/nextcloud/text/issues/2836 into consideration should we take action for this one right now and instead of adding this as yet another menu bar entry try to group them in the callout menu as listed in the issue? 

>  Blockquote and code block go into the callouts menu

In the end the details is just another "block style"

Reply:
Though one argument against would be that you could still have a callout nested in a details block which is then not very obvious from the active indication in the menu bar

Reply:
> In the end the details is just another "block style"

Yeah, good point, makes sense to directly group it properly.

Reply:
I agree, but then we probably should move block quote and code blocks into this submenu as well straight away, right? What icon should be used for the "block style" submenu?